app/controllers/authenticated/app-monitoring/monitoring-details.ts

```diff
@@ -0,0 +1,6 @@
+import Controller from '@ember/controller';
+import AmAppModel from 'irene/models/am-app';
+
+export default class AuthenticatedAppMonitoringMonitoringDetailsController extends Controller {
+  declare model: AmAppModel;
```

Comment:
whats the purpose of these controllers ?

Reply:
Originally this was added for typing purposes at the template files. It is no longer required. They would be deleted.